.github/workflows/regression-tests.yml

```diff
@@ -55,9 +55,6 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_S3_ACCESS_KEY }}
           aws-secret-access-key: ${{ secrets.AWS_S3_ACCESS_SECRET }}
           s3-bucket: ${{ secrets.S3_REGTEST_BUCKET }}
-          # Chain ternary oprator of the form (which can be nested)
-          # (expression == condition && <true expression> || <false expression>)
-          epoll: ${{ matrix.proactor == 'Epoll' && 'epoll' || 'iouring' }}
```

Comment:
Why did you remove this ?

Reply:
why it is needed if it's for iouring only?

Reply:
Because we use `inputs.epoll` to print. It will be `empty` in your case. And also we do `if [[ "${{inputs.epoll}}" == 'epoll' ]];` and an unconditional `else` for `iouring`. If we change this, we will break it without noticing.

Not important though, I will sent a patch